CombinatorialGames/Game/Loopy/Outcome.lean

```diff
@@ -0,0 +1,237 @@
+import CombinatorialGames.Game.Loopy.IGame
+
+namespace LGame
+
+/-- A surviving strategy for Left, going second.
+
+This is a set of states, such that for every move Right makes, Left can bring it back
+to the set. -/
+def LeftStrategy (s : Set LGame) : Prop :=
+  ∀ y ∈ s, ∀ z ∈ y.rightMoves, ∃ r ∈ z.leftMoves, r ∈ s
+
+/-- A surviving strategy for Right, going second.
+
+This is a set of states, such that for every move Left makes, Right can bring it back
+to the set. -/
+def RightStrategy (s : Set LGame) : Prop :=
+  ∀ y ∈ s, ∀ z ∈ y.leftMoves, ∃ r ∈ z.rightMoves, r ∈ s
+
+@[simp]
+theorem leftStrategy_neg {s : Set LGame} : LeftStrategy (-s) ↔ RightStrategy s := by
+  simp [LeftStrategy, RightStrategy]
+
+@[simp]
+theorem rightStrategy_neg {s : Set LGame} : RightStrategy (-s) ↔ LeftStrategy s := by
+  simp_rw [← leftStrategy_neg, neg_neg]
+
+/-- `IsLeftSurviving x` means that left survives `x` going second. -/
+def IsLeftSurviving (x : LGame) : Prop := ∃ s : Set LGame, x ∈ s ∧ LeftStrategy s
+
+/-- `IsRightSurviving x` means that right survives `x` going second. -/
+def IsRightSurviving (x : LGame) : Prop := ∃ s : Set LGame, x ∈ s ∧ RightStrategy s
+
+theorem leftStrategy_iUnion {ι} {s : ι → Set LGame} (h : ∀ i, LeftStrategy (s i)) :
+    LeftStrategy (⋃ i, s i) :=
+  fun y hy z hz ↦ have ⟨i, hi⟩ := Set.mem_iUnion.mp hy
+    have ⟨r, hrz, hr⟩ := h i y hi z hz
+    ⟨r, hrz, Set.mem_iUnion_of_mem i hr⟩
+
+theorem rightStrategy_iUnion {ι} {s : ι → Set LGame} (h : ∀ i, RightStrategy (s i)) :
+    RightStrategy (⋃ i, s i) :=
+  fun y hy z hz ↦ have ⟨i, hi⟩ := Set.mem_iUnion.mp hy
+    have ⟨r, hrz, hr⟩ := h i y hi z hz
+    ⟨r, hrz, Set.mem_iUnion_of_mem i hr⟩
```

Comment:
These two are currently unused; I also removed two lemmas that have simple proofs:
```
example {x : LGame} :
    IsLeftSurviving x ↔ ∀ y ∈ x.rightMoves, ∃ r ∈ y.leftMoves, IsLeftSurviving r := by
  rw [isLeftSurviving_iff_forall]
  simp_rw [not_isRightWinning_iff_exists]

example {x : LGame} :
    IsRightSurviving x ↔ ∀ y ∈ x.leftMoves, ∃ r ∈ y.rightMoves, IsRightSurviving r := by
  rw [isRightSurviving_iff_forall]
  simp_rw [not_isLeftWinning_iff_exists]
```